examples/concurrency/lib/simple_isolate_closure.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:convert';
+import 'dart:io';
+import 'dart:isolate';
+
+const String filename = 'with_keys.json';
+
+// #docregion main
+void main() async {
+  // Read some data.
+  final jsonData = await Isolate.run(() {
```

Comment:
There should be an `async` at `() async {`, to make it possible to use `await` in the function.